src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How The Merge impacts ether supply
+description: Breakdown on how The Merge will impact ether supply
```

Comment:
We should consistently try to use ether or ETH on ethereum.org. I feel like ETH makes more sense. 

```suggestion
description: A breakdown of how The Merge will impact ETH supply
```